src/nrtk_explorer/app/transforms.py

```diff
@@ -153,6 +154,9 @@ def delete_meta_state(old_ids, new_ids):
             self._transforms["nrtk_blur"] = nrtk_trans.NrtkGaussianBlurTransform()
             self._transforms["nrtk_pybsm"] = nrtk_trans.NrtkPybsmTransform()
 
+        # Add transform from YAML definition
+        self._transforms.update(nrtk_yaml.generate_transforms())
```

Comment:
Erotemic tried to run once without `nrtk`, https://github.com/Kitware/nrtk-explorer/pull/91
Not sure why we want to support that, but there are checks for `nrtk_transforms_available` above.  I moved the line under the if statment in following PR.

Reply:
no need, the code already take care of that and skip any definition when the package is not found.